JSDemos/Demos/Charts/Strips/description.md

```diff
@@ -0,0 +1,5 @@
+A strip is a colored piece of the **Chart**'s background that highlights a range of values.
+
+To configure strips, declare an array of objects in the **argumentAxis**.[strips[]](Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/strips/) or **valueAxis**.[strips[]](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/strips/) property. Each object in this array configures an individual strip. The [startValue](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/strips/#startValue) and [endValue](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/strips/#endValue) properties define limits of the area which a strip occupies. This demo shows two strips on the value axis. We configure the [label](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/strips/label/)'s [text](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/strips/label/#text) and **font**.[color](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/strips/label/font/#color) properties for each strip separately.
+
+You can specify the same appearance for all strips on a specific axis or in the **Chart**. To do this, declare the [stripStyle](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/stripStyle/) object in the [valueAxis](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/), [argumentAxis](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/), or [commonAxisSettings](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/) object. Individual settings override common settings. In this demo, we use the **stripStyle** object to specify the font [weight](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/stripStyle/label/font/#weight) and [size](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/stripStyle/label/font/#size) for all strip labels on the value axis. 
```

Comment:
A specific axis is also in the Chart. That's why
```suggestion
You can specify the same appearance for strips on a specific axis or all axes. To do this, declare the [stripStyle](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/stripStyle/) object in the [valueAxis](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/), [argumentAxis](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/), or [commonAxisSettings](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/) object. Individual settings override common settings. In this demo, we use the **stripStyle** object to specify the font [weight](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/stripStyle/label/font/#weight) and [size](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/stripStyle/label/font/#size) for all strip labels on the value axis. 
```

Reply:
I guess that Roman's logic in this sentence was the following: same appearance both for all strips on an axis and in the Chart.